web_src/less/_base.less

```diff
@@ -97,6 +97,7 @@
   --color-input-border-hover: #cecece;
   --color-navbar: #f8f8f8;
   --color-light: #00000004;
+  --color-light-undo-disabling: rgba(0, 0, 0, calc(4 / 255 * 222 / 255 / var(--opacity-disabled)));
```

Comment:
I don't quite understand this. What is the result of the calc?

Reply:
222 / 255 = .87, a constant value, seen quite often in e. g. Fomantic.
4 / 255 is of course the original value right above this.
So this basically restores the opacity (times its reciprocal) and then times the constant factor .87 to make the contrast less stark.
The current setting would compile this to the needed decimal value. This is just to avoid rounding errors.